tests/app/template/test_rest.py

```diff
@@ -449,8 +419,8 @@ def test_should_be_error_on_update_if_no_permission(
     assert json_resp['message'] == expected_error
 
 
-@pytest.mark.xfail(reason='Mislabelled for route removal, fails when unskipped', run=False)
-# def test_should_error_if_created_by_missing(client, sample_user, sample_service):
+@pytest.mark.skip(reason='TODO #2336 - Fail due to orphaned User object')
+@pytest.mark.serial
```

Comment:
not serial

Reply:
Updated and remove skip.

Confirm test passes locally and on Github Jobs.